### ISSUE TITLE
first evolutionary training param exploration

### ISSUE THREAD

**Issue by leandermasopust** [Deep-Learning-as-a-Service/EvA#4]
- [ ] write a simple evolutionary algorithm that takes a given model and given processed, normalized data and generates a set of locally optimal hyperparamaters such as learning rate, batch size etc.
